src/ParametricOptInterface.jl

```diff
@@ -261,11 +280,31 @@ function MOI.supports(
     return MOI.supports(model.optimizer, attr, tp)
 end
 
-# TODO
-# This is not correct, you need to put the parameters back into the function
-# function MOI.get(model::ParametricOptimizer, attr::MOI.ConstraintFunction, ci::MOI.ConstraintIndex{F, S}) where {F, S}
-#     MOI.get(model.optimizer, attr, ci)
-# end
+function MOI.get(
+    model::ParametricOptimizer,
+    attr::MOI.ConstraintFunction,
+    ci::MOI.ConstraintIndex{F,S},
+) where {F,S}
+    # Check if the index was cached as Affine expression
+    if !isempty(model.affine_constraint_cache)
+        haskey(model.affine_constraint_cache, ci) &&
+            return model.affine_constraint_cache[ci]
+        # Check if the index was cached as a Quadratic
+    elseif !isempty(model.quadratic_added_cache)
+        haskey(model.quadratic_added_cache, ci) &&
+            return model.quadratic_added_cache[ci]
+    else
+        return MOI.get(model.optimizer, attr, ci)
+    end
+end
```

Comment:
This function is not correct. See https://github.com/jump-dev/MathOptInterface.jl/issues/1522#issuecomment-896391654

Reply:
arrrg I left the `isempty`. Fixed in e1374fa